tests/testdata/mock-java-original-toc.yml

```diff
@@ -0,0 +1,299 @@
+### YamlMime:TableOfContent
+- uid: "com.google.cloud.memcache.v1beta2.stub"
+  name: "com.google.cloud.memcache.v1beta2.stub"
+  items:
+  - uid: "com.google.cloud.memcache.v1beta2.stub.CloudMemcacheStub"
+    name: "CloudMemcacheStub"
```

Comment:
Would you mind shortening this test file so it's easier to see what exactly changes?